vision/cloud-client/detect/vision_batch_annotate_files_gcs.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_batch_annotate_files_gcs]
+
+from google.cloud import vision_v1
+from google.cloud.vision_v1 import enums
+
+
+def sample_batch_annotate_files(
+    storage_uri="gs://cloud-samples-data/vision/document_understanding/kafka.pdf",
```

Comment:
nit: Is this a publicly accessible bucket or should this be in the something like `"gs://YOUR-PROJECT-ID/path/to/your/file.pdf"`

Reply:
Yes, this is publicly accessible, confirmed with a non-corp account.